### PR TITLE
Update stats!!

### PR DESCRIPTION
**Pull request by plaindocs**
- [x] Apply to Australia

### CODE REVIEW

File: docs/conf/prague/2021/press-kit.rst [hunk 1]
```diff
@@ -38,9 +38,8 @@ Useful numbers
 
 We have some stats about attendees, membership, and website hits over on the following blog posts:
 
-- :doc:`/blog/write-the-docs-2017-stats/`
-- :doc:`/blog/write-the-docs-2016-year-in-review/`
-
+- :doc:`/blog/2021-january-community-update/`
+- :doc:`/blog/write-the-docs-2020-stats/`
```
Comment (ericholscher) [inline docs/conf/prague/2021/press-kit.rst]: We should make this an include across all conferences :)

Comment (plaindocs) [reply docs/conf/prague/2021/press-kit.rst]: That does make sense.

**Review by ericholscher — APPROVED** [review]
Looks good. We should also apply it to Portland, no?

**Comment by plaindocs** [comment]
Sure, I care more about next year than this year, but sure.

**Pull request merged**